convex/accessControl.ts

```
@@ -0,0 +1,565 @@
+  reason?: string;
+  userId?: string;
+  resourceId?: string;
+}
+
+/**
+ * Session validation result
+ */
+interface SessionValidation {
+  valid: boolean;
+  userId?: string;
+  sessionData?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)